cookbook/security/user_checkers.rst

```diff
@@ -0,0 +1,215 @@
+.. index::
+    single: Security; Creating and Enabling Custom User Checkers
+
+How to Create and Enable Custom User Checkers
+=============================================
+
+During the authentication of a user, additional checks might be required to verify
+if the identified user is allowed to log in. By defining a custom user checker, you
+can define per firewall which checker should be used.
+
+.. versionadded:: 2.8
+    Defining a custom user checker was introduced in Symfony 2.8.
+
+
+Creating a Custom User Checker
+------------------------------
+
+User checkers are defined in PHP classes that must implement the
+:class:`UserCheckerInterface Symfony\\Component\\Security\\Core\\UserCheckerInterface`.
```

Comment:
USerCheckInterface in front should be removed, it will be automatically added. (```:class:`Symfony\\Component\\Security\\Core\\UserCheckerInterface` ```)